src/main/java/org/sonarlint/intellij/trigger/MakeTrigger.java

```diff
@@ -55,7 +55,7 @@ public void beforeBuildProcessStarted(Project project, UUID sessionId) {
   }
 
   @Override public void buildFinished(Project project, UUID sessionId, boolean isAutomake) {
-    if (!project.equals(myProject) || !isAutomake) {
+    if (project == null || !project.equals(myProject) || !isAutomake) {
```

Comment:
![MAJOR](https://raw.githubusercontent.com/SonarCommunity/sonar-github/master/images/severity-major.png) Change this condition so that it does not always evaluate to "false" [![rule](https://raw.githubusercontent.com/SonarCommunity/sonar-github/master/images/rule.png)](https://next.sonarqube.com/sonarqube/coding_rules#rule_key=squid%3AS2583)